tools/p3_compatibility_check

```diff
@@ -0,0 +1,6 @@
+python-modernize --enforce \
```

Comment:
Maybe better to spell out the file name as python3_compatibility_check? May not be obvious what p3 means.